packages/grafana-data/src/utils/url.ts

```diff
@@ -129,3 +131,24 @@ export const urlUtil = {
   appendQueryToUrl,
   getUrlSearchParams,
 };
+
+export function serializeStateToUrlParam(urlState: ExploreUrlState, compact?: boolean): string {
```

Comment:
Also this needed to be moved here because of cyclic imports.

Reply:
Can we be more explore-especific here in naming: `serializeStateToUrlParam` -> `serializeExploreStateToUrlParam`? I guess this will be brought back to core when we refactor.